src/app/Notifications/Notifications.tsx

```diff
@@ -46,10 +46,18 @@ export interface Notification {
   key?: string;
   title: string;
   message?: string;
+  category?: string;
   variant: AlertVariant;
   timestamp?: number;
 }
 
+export interface NotificationDrawerCategory {
```

Comment:
I would probably move this into the same file as the component where it's used - it doesn't seem quite related to this service, which is more about the pure information side of a notification and not so much about the presentation of them.